src/seaport.ts

```diff
@@ -2881,8 +2881,8 @@ export class OpenSeaPort {
     if (priceDiff > 0 && expirationTime == 0) {
       throw new Error('Expiration time must be set if order will change in price.')
     }
-    if (englishAuctionReservePrice && (startAmount < englishAuctionReservePrice)) {
-      throw new Error('Reserrve price must be greater than or equal to the start amount.')
+    if (englishAuctionReservePrice && (englishAuctionReservePrice < startAmount)) {
```

Comment:
should have mentioned this earlier but we should also check that `waitingForBestCounterOrder` is true if you have a `englishAuctionReservePrice` set

Reply:
also yay the tests caught a big bug?